Application/EdFi.Ods.Admin.Api/Compose/pgsql/compose-build.yml

```diff
@@ -15,40 +15,38 @@ services:
     ports:
       - "443:443"
       - "80:80"
-    container_name: ed-fi-adminapi-gateway
+    container_name: ed-fi-gateway-adminapi-packaged
     restart: always
     hostname: nginx
     volumes:
       - ../../Docker/ssl:/ssl/
+      - ../../Docker/gateway/adminapi-packaged.conf:/etc/nginx/templates/default.conf.template
```

Comment:
@stephenfuqua 
We already have "default.conf.template" file for nginx configuration on gateway folder. Should we consider using this file instead !?

Reply:
The `default.conf.template` file has a configuration for serving the ODS/API. This compose file doesn't start the ODS/API, and then NGiNX throws an error and is unusable. So we need a different conf file that only has AdminAPI, without the ODS/API.